index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <link rel="stylesheet" href="styles.css">
+        <title>first mobile app</title>
+    </head>
+
+    <body>
+        <section class="main-page">
+            <header class="main-header">
+                <nav>
+                    <div class="header-text">John Doe</div>
+                </nav>
+                <nav class="union">
+                    <div class="vector"></div>
+                    <div class="vector"></div>
+                    <div class="vector"></div>
+                </nav>
+            </header>
+            
+        <div class="page1">
+            <h1 class="main-title">Hey There.<br>I'm Alex</h1>
+            <h2 class="job-title">I'm a Software Developer</h2>
+            <h5 class="regular-text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
```

Comment:
- [ ] This should be a p tag since it describes. This way will be semantically correct and the screen readers and other technologies will thank you

Reply:
@Olamarx Hello, thanks for sparing me time, you mean the h5 tag should be p tag? am I correct?

Reply:
Yes